BookingService/BookingService.Application/UserService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BookingService.Application.DTOs;
+using BookingService.Domain;
+using Microsoft.AspNetCore.Identity;
+
+namespace BookingService.Application
+{
+    public class UserService
+    {
+        private ITokenService _tokenService;
+        private readonly UserManager<User> _userManager;
+        private readonly SignInManager<User> _signInManager;

```

Comment:
Create custom wrappers over these classes, because u will not be able to mock them. It's 5th solid principle DI - classes should not be dependent on other classes, but should be dependent on abstractions (interfaces)